astropy/io/fits/tests/test_image.py

```diff
@@ -1010,7 +1010,7 @@ def test_comp_image_hcompress_image_stack(self):
         hdu.writeto(self.temp('test.fits'))
 
         with fits.open(self.temp('test.fits')) as hdul:
-            assert (hdul['SCI'].data == cube).all()
+            assert np.abs(hdul['SCI'].data - cube).max() < 1./15.
```

Comment:
Maybe also specifically mention in a comment what you wrote in the issue concerning 1 / quantize_level.  Otherwise this looks a bit mysterious.